src/FakeItEasy.Net35/Compatibility/ConcurrentQueue.cs

```diff
@@ -0,0 +1,34 @@
+namespace FakeItEasy

```

Comment:
I'm not wild about the namespace, but if I use System.Collection.Concurrent, the compiler can't tell it from the real one when it builds the .NET 3.5 unit tests. Same as for `ConditionalWeakTable`.
An alternative would be to not have any .NET 3.5 tests for this class and add tests that would verify that the bug is fixed (like `Should_not_throw_when_asserting_while_calls_are_being_made_on_the_fake` below, but to run against the 3.5 library). I actually kind of prefer that, but if we're going to take that step would really like to pull in _all_ the tests (and specs) to be run against all versions of the library, and that (should) get easier with shared projects in VS2015, so wonder if we shouldn't just drag our feet a little.
Regardless, as always, I'll do whatever you like!